moped-editor/src/utils/mapHelpers.js

```diff
@@ -635,15 +632,15 @@ export const createProjectViewLayerConfig = (
  */
 export const renderTooltip = (tooltipText, hoveredCoords, className) =>
   tooltipText && (
-    <div
+    <span
       className={className}
       style={{
         left: hoveredCoords?.x,
         top: hoveredCoords?.y,
       }}
     >
-      <div>{tooltipText.toLowerCase()}</div>
-    </div>
+      <span>{tooltipText.toLowerCase()}</span>
+    </span>
```

Comment:
Really liking these changes, it never occurred to me to try using a span tag instead. 🙌